src/App.tsx

```diff
@@ -21,14 +37,20 @@ export const App: React.FC = () => {
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {(loading && !currentTodo) && (
+                <Loader />
+              )}
+              {todos.length !== 0 && (
```

Comment:
```suggestion
              {!!todos.length && (
```
**PLS DO THE SAME FOR ALL SUCH CASES IN THE PROJECT**